doc/api.md

```diff
@@ -75,26 +75,26 @@ information configured:
+| `cbHost`                       | `cbHost`                       | Context Broker connection information. This options can be used to override the global ones for specific types of devices.                                                                                                                                                |
+| `lazy`                         | `lazy`                         | list of common lazy attributes of the device. For each attribute, its `name` and `type` must be provided.                                                                                                                                                                 |
+| `commands`                     | `commands`                     | list of common commands attributes of the device. For each attribute, its `name` and `type` must be provided, additional `metadata` is optional.                                                                                                                          |
+| `attributes`                   | `attributes`                   | list of common active attributes of the device. For each attribute, its `name` and `type` must be provided, additional `metadata` is optional.                                                                                                                            |
+| `static_attributes`            | `staticAttributes`             | this attributes will be added to all the entities of this group 'as is', additional `metadata` is optional.                                                                                                                                                               |
+| `internal_attributes`          | `internalAttributes`           | optional section with free format, to allow specific IoT Agents to store information along with the devices in the Device Registry.                                                                                                                                       |
+| `expressionLanguage`           | `expresionLanguage`            | optional boolean value, to set expression language used to compute expressions, possible values are: legacy or jexl. When not set or wrongly set, `legacy` is used as default value.                                                                                      |
+| `explicitAttrs`                | `explicitAttrs`                | optional boolean value, to support selective ignore of measures so that IOTA doesn’t progress. If not specified default is `false`. Additionally `explicitAttrs` can be used to define which meassures defined in JSON/JEXL array will be propagated to NGSI interface.   |
```

Comment:
I suggest to add a link to advanced-topics.md#explicitly-defined-attributes-explicitattrs at this point, for a more detailed explanation of explicitAttrs functionality.

Reply:
In addition, I understand that the explicitAttrs entry at the device API table should be also adjusted in a similar way.

Reply:
done in https://github.com/telefonicaid/iotagent-node-lib/pull/1180/commits/cd1fe6af5fdad079b750971b365646ec4ecbcd8b